src/lib/y2network/sysconfig/config_writer.rb

```diff
@@ -77,7 +78,8 @@ def write_interface_changes(config, old_config)
           file.remove
         end
 
-        nil
+        write_dns_settings(config, old_config)
```

Comment:
so we do not write dns previously?

Reply:
or better, why it have to be here and not in def write? can you document reason? as I see it written twice

Reply:
Good catch. It was a problem with merging. I will fix it.